packages/mux-player/src/index.ts

```diff
@@ -361,6 +361,17 @@ class MuxPlayerElement extends VideoApiElement {
 
     // toggles activeCues for a particular track depending on whether the user is active or not
     const toggleLines = (track: TextTrack, userInactive: boolean) => {
+      // if we're live, exit early from here because our controls design means captions don't need to shift
+      // unless secondary color is set, in which case the entire bottom control bar may be opaque
+      if (
+        this.streamType &&
+        ['live', 'll-live'].includes(this.streamType) &&
+        !this.secondaryColor &&
+        this.offsetWidth >= 800
```

Comment:
seems a very specific condition to me, is it worth to disable instead of keeping it uniform? 
I'm leaning to keep it uniform unless the experience is a lot better this way

Reply:
it is a bit. If the player is smaller than ~800px there's a higher chance that the volume slider will overlap captions but when it's larger it becomes less likely.